src/services/api/graphql/generate/schema_from_config_session.py

```diff
@@ -29,50 +29,23 @@
     sys.path.append("/usr/libexec/vyos/services/api")
```

Comment:
Another instance of a hardcoded path. I don't expect that we change it, but if switching distros will make us move it to elsewhere, storing all such constants in one place will make it a lot easier.

Reply:
As above.